pkg/maps/policymap/policymap.go

```diff
@@ -84,6 +84,36 @@ type PolicyEntry struct {
 	Bytes     uint64 `align:"bytes"`
 }
 
+// CallKey is the index into the prog array map.
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapKey
+type CallKey struct {
+	index uint32
+}
+
+// CallValue is the program ID in the prog array map.
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapValue
+type CallValue struct {
+	progID uint32
+}
+
+// GetKeyPtr returns the unsafe pointer to the BPF key
+func (k *CallKey) GetKeyPtr() unsafe.Pointer { return unsafe.Pointer(k) }
+
+// GetValuePtr returns the unsafe pointer to the BPF value
+func (v *CallValue) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(v) }
+
+// String converts the key into a human readable string format.
+func (k *CallKey) String() string { return fmt.Sprintf("%d", k.index) }
```

Comment:
Small nit but these returns could easily be `strconv.Itoa(k.index)`

Reply:
`k.index` is `unit32` here so unless there's a preferred way, I'll change to the following:
```go
strconv.FormatUint(uint64(k.index), 10)
```